SU-HSDO/suhumsci
# SHS-5626: Implementation: New Accordion Style (FED)

# READY FOR REVIEW

## Summary
Add new implementation to new light variation to the Accordion 

## Need Review By (Date)
16/04

## Urgency
medium

## Steps to Test
1. Create a Flexible Page on Colorful and Traditional
2. Add an Accordion component and select the "Light" variation on styles
3. Confirm it looks like Option C (for [Traditional](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1087-199&t=KlSFB3CTMJaSLLXe-4) - for [Colorful](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1042-302&t=KlSFB3CTMJaSLLXe-4)) on Figma
4. Please change the appereance and try this new Accordion variation on different color pairings

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
